include/psp2/appmgr.h

```diff
@@ -67,11 +67,15 @@ typedef struct SceAppMgrSystemEvent {
 
 typedef struct SceAppMgrAppState SceAppMgrAppState; // Missing struct
 typedef struct SceAppMgrExecOptParam SceAppMgrExecOptParam; // Missing struct
+typedef struct SceAppMgrLaunchAppOptParam SceAppMgrLaunchAppOptParam; // Missing struct
 
 #define SCE_APPMGR_MAX_APP_NAME_LENGTH	(31)
 
 //! name: The Title ID of the application
-int _sceAppMgrDestroyAppByName(char *name);
+int sceAppMgrDestroyAppByName(char *name);
+
+//! name: The Title ID of the application
+int _sceAppMgrDestroyAppByName(char *name, char temp[3]);
```

Comment:
Nearly every _sceAppMgr has got a temporary work buffer which are passed in the wrapper module SceDriverUser. Using sceAppMgrDestroyAppByName wouldn't require to set this temporary buffer. Idecided to still keep it, because the latest VitaShell uses this API.